src/SFA.DAS.ApprenticeCommitments/Application/Services/ApprenticeCommitmentsService.cs

```diff
@@ -0,0 +1,43 @@
+using SFA.DAS.ApprenticeCommitments.Configuration;
+using SFA.DAS.ApprenticeCommitments.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Application.Services
+{
+    public class ApprenticeCommitmentsService //: IEmployerIncentivesService
```

Comment:
Do we not need the interface here

Reply:
Once there is a need for the interface then yes.  As of the state of the project right now there is no need for one, is there?